src/ng/filter/orderBy.js

```diff
@@ -599,7 +602,7 @@ function orderByFilter($parse) {
         }
       }
 
-      return compare(v1.tieBreaker, v2.tieBreaker) * descending;
+      return (compare(v1.tieBreaker, v2.tieBreaker) || defaultCompare(v1.tieBreaker, v2.tieBreaker)) * descending;
```

Comment:
I do not fully understand this change, this should be just
`return defaultCompare(v1.tieBreaker, v2.tieBreaker) * descending;`

There is no reason for an external function to understand the internal `tieBreaker` structure.